webapp/src/Service/DOMJudgeService.php

```diff
@@ -628,6 +637,15 @@ public function getDomjudgeWebappDir(): string
         return $this->params->get('domjudge.webappdir');
     }
 
+    /**
+     * Get the directory used for storing cache files
+     * @return string
+     */
+    public function getCacheDir(): string
```

Comment:
For my understanding: where is this used? Also, how does the cacheDir get passed to the config service constructor?

Reply:
Ah sorry, it's not used anymore. In the beginning I had ConfigurationService use DOMJudgeService, but that was not possible since DOMJudgeService needs ConfigurationService to read some config, so I injected `$cacheDir` directly (see the `services.yaml` diff)